lfshttp/cookies.go

```diff
@@ -0,0 +1,20 @@
+package lfshttp
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/google/slothfs/cookie"
```

Comment:
I assume we need this dependency because the builtin Go cookie jar can't read from a file?

Reply:
Yes. Git-lfs should read the same file as git and libcurl - a parser for Netscape cookie jar file format is needed (https://curl.haxx.se/docs/http-cookies.html).

The format is quite simple and the parser can be added directly to `lfshttp/cookies.go` if you would like to avoid additional dependencies.